<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimeWidget</class>
 <widget class="QWidget" name="TimeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>60</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>60</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>60</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>12</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>12</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="CellsWidget" name="widgetYears" native="true"/>
   </item>
   <item>
    <widget class="CellsWidget" name="widgetMonths" native="true"/>
   </item>
   <item>
    <widget class="CellsWidget" name="widgetDays" native="true"/>
   </item>
   <item>
    <widget class="CellsWidget" name="widgetHours" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CellsWidget</class>
   <extends>QWidget</extends>
   <header>timewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
